string[] names

int32[] values

---
CameraSettings new_settings
